app/components/DocumentBreadcrumb.js

```diff
@@ -58,18 +58,9 @@ function useCategory(document) {
 
 const DocumentBreadcrumb = ({ document, children, onlyText }: Props) => {
   const { collections } = useStores();
-  const { t } = useTranslation();
   const category = useCategory(document);
 
-  let collection = collections.get(document.collectionId);
```

Comment:
This is interesting, I think that the Deleted Collection logic is still needed, but needs to be modified to check `document.collectionId` exists – after all docs in trash with deleted collections are still a thing.